.github/workflows/test.yml

```diff
@@ -41,6 +42,45 @@ jobs:
       run: pip install -r requirements-dev.txt
     - name: Run tests
       run: pytest
+  check_db:
+    name: Check DB migrations
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        # Docker Hub image
+        image: postgres:14.5
+        # Provide the password for postgres
+        env:
+          POSTGRES_PASSWORD: password
+          POSTGRES_DB: data_store_test
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          # Maps tcp port 5432 on service container to the host
+          - 5432:5432
+      localstack:
+        image: localstack/localstack:3.2
+        env:
+          # these match the values in config/unit_test.py
+          AWS_REGION: eu-central-1
+          AWS_ACCESS_KEY_ID: test
+          AWS_SECRET_ACCESS_KEY: test
+        ports:
+          - 4566:4566
+          - 4571:4571
+        options: --health-cmd="curl http://localhost:4566/health" --health-interval=10s --health-timeout=10s --health-retries=10
```

Comment:
Do we actually need localstack for this check?

Reply:
let's check 👀